index.js

```diff
@@ -81,3 +81,39 @@ document.addEventListener('DOMContentLoaded', () => {
     books.counter += 1;
   });
 });
+// Clock
+const dateElement = document.getElementById('date');
+const localdate = () => {
+  const date = new Date();
+  const dateString = date.toLocaleString();
+  dateElement.textContent = dateString;
+};
+
+setInterval(localdate, 1000);
+// End of Clock
+// WEB APP
+const listBtn = document.querySelector('#list-btn');
+const addBtn = document.getElementById('add-btn');
+const contactBtn = document.getElementById('contact-btn');
+const texttest = document.getElementById('add_book');
+const contact = document.getElementById('contact');
+const addForm = document.getElementById('add-form');
+
+listBtn.addEventListener('click', () => {
+  list.className = 'display';
+  contact.className = 'contact nodisplay';
+  addForm.className = 'display-form nodisplay';
+  texttest.innerText = 'All awesome books';
+});
+contactBtn.addEventListener('click', () => {
+  list.className = 'nodisplay';
+  contact.className = 'contact display';
+  addForm.className = 'display-form nodisplay';
+  texttest.innerText = 'Contact Information';
+});
+addBtn.addEventListener('click', () => {
+  list.className = 'nodisplay';
+  contact.className = 'contact nodisplay';
+  addForm.className = 'display-form display';
+  texttest.innerText = 'Add a new book';
+});
```

Comment:
- [ ] I can't navigate the site, kindly fix this so your app can have 3 sections that are appropriately displayed when a user clicks on the nav links

![Screenshot_20220805_070945](https://user-images.githubusercontent.com/25227022/183013256-1a33a5e1-95b8-47ec-86bf-00d1ae04a6fb.png)